src/sotoki/utils/database/posts.py

```diff
@@ -49,7 +49,7 @@ def questions_key():
 
     @staticmethod
     def questions_stats_key():
-        return "nb_answers"
+        return "questions_stats"
```

Comment:
Was this a typo or did you change it?

Reply:
I changed this because is was disturbing to have a JSON object named `nb_answers` inside Redis, with an `nb_answers` (and others) properties inside. Looked like a legacy never updated. And I don't expect this change to have any impact, but I might be wrong ^^